File: MbUnit/Saucery/Capabilities/ConcreteCreators/WebDriverIOSCreator.cs
using System;
using Saucery.Capabilities.Base;
using Saucery.Capabilities.ConcreteProducts;
using Saucery.TestDataSources;

namespace Saucery.Capabilities.ConcreteCreators {
    internal class WebDriverIOSCreator : Creator {
        public override BaseCapabilities Create(PlatformTestData platform, String testName, string nativeApp = null, bool useChromeOnAndroid = false) {
            return new WebDriverIOSCapabilities(platform, testName);
        }
    }
}
/*
 * Copyright Andrew Gray, Full Circle Solutions
 * Date: 18th September 2014
 * 
 */

File: JUnit/saucery-core/src/main/java/au/com/fullcirclesolutions/saucery/capabilities/WebDriverIOSCreator.java
package au.com.fullcirclesolutions.saucery.capabilities;

class WebDriverIOSCreator extends Creator {
    @Override
    public BaseCapabilities Create(SaucePlatform platform, String testName) {
        return new WebDriverIOSCapabilities(platform, testName);
    }
}
/*
 * Copyright: Andrew Gray, Full Circle Solutions
 * Date: 18th September 2014
 */